apps/omg_watcher/lib/omg_watcher/exit_processor/exit_info.ex

```diff
@@ -137,4 +147,21 @@ defmodule OMG.Watcher.ExitProcessor.ExitInfo do
   # `exitable` will be `false` ALONG WITH the whole tuple holding zeroees, if the exit was processed successfully
   # **NOTE** one can only rely on the zero-nonzero of this data, since for processed exits this data will be all zeros
   defp parse_contract_exit_status({exitable, _, _, _, _, _}), do: exitable
+
+  @doc """
+  Based on the block number determines whether UTXO was created by a deposit.
+  """
+  defguard is_deposit(blknum) when rem(blknum, @child_block_interval) != 0
+
+  @doc """
+  Calculates the time at which an exit can be processed and released if not challenged successfully.
+  See https://docs.omg.network/challenge-period for calculation logic.
+  """
+  @spec calculate_sft(pos_integer(), pos_integer(), pos_integer(), pos_integer()) :: {:ok, pos_integer()}
+  def calculate_sft(blknum, exit_timestamp, utxo_creation_timestamp, min_exit_period)
+      when is_deposit(blknum),
+      do: {:ok, max(exit_timestamp + min_exit_period, utxo_creation_timestamp + min_exit_period)}
+
+  def calculate_sft(_blknum, exit_timestamp, utxo_creation_timestamp, min_exit_period),
+    do: {:ok, max(exit_timestamp + min_exit_period, utxo_creation_timestamp + 2 * min_exit_period)}
```

Comment:
these are multiline functions, why squash them into a single line function defintion? use `do...end`

Reply:
Refactored in dd9ac6b.